antidox/perspective.py

```diff
@@ -106,36 +110,61 @@ def contains_toxicity(perspective_response):
   return is_toxic
 
 
+def use_query(client, sql_query):
+  """make big query api request"""
+  query_job = client.query(sql_query)
+  rows = query_job.result()
+  strlst = []
+  for row in rows:
+    strlst.append(str(row.cleaned_content))
+  return strlst
+  # query_job = big_q.query("""SELECT cleaned_content FROM
```

Comment:
Can you remove these commented out lines, and other commented out lines that appear later in the file?  Ideally any comments we keep should be full sentences that explain what's going on in the code

Reply:
done